src/scheduler/data_types.h

```diff
@@ -176,6 +177,19 @@ typedef sch_resource_t usage_t;
 typedef void event_ptr_t;
 typedef int (*event_func_t)(event_ptr_t*, void *);
 
+typedef struct sched_cmd {
+	int cmd;   /* sched command */
+	char *jid; /* jobid assisiated with cmd if any else NULL */
+} sched_cmd_t;
+
+typedef struct svr {
+	char *svrhost;
+	int pfd;		     /* primary connection to server - used for IFL calls */
+	int sfd;		     /* secondary connection to server - used to receive sched cmds from server */
+	sched_cmd_t cmd;	     /* command received from server */
+	pbs_list_link all_svrs_link; /* link to all servers */
```

Comment:
We can just name it as next_svr_link.

Reply:
why? its link to all svrs list...

Reply:
This is because it only has the link for next server. That link will again have a link for further servers. Look at other places for example structure pbs_sched also we only call the link as sc_link we don't call this as all_sc_link.

Reply:
pbs_list_head and pbs_list_link is doubly link list, and this link can be used to find next as well as prior also... and in my view sc_link in pbs_sched is not correct it should be all_sc_link only, see in job->ji_alljobs in job struct.